product_docs/docs/pgd/5/quickstart/index.mdx

```diff
@@ -7,44 +7,57 @@ indexCards: none
 navigation:
 - quick_start_docker
 - quick_start_aws
+- further_explore_conflicts
 ---
 
 ## Quick Start
 
-Postgres Distributed (PGD) is a multi-master replicating implementation of Postgres designed for high performance and availability. You can create database clusters comprising of many bidirectionally synchronizing database nodes. The clusters can have a number proxy servers that direct your query traffic to the most available nodes, adding further resilience to your cluster configuration.
+Postgres Distributed (PGD) is a multi-master replicating implementation of Postgres designed for high performance and availability. You can create database clusters made up of of many bidirectionally synchronizing database nodes. The clusters can have a number of proxy servers that direct your query traffic to the most available nodes, adding further resilience to your cluster configuration.
  
-PGD is very configurable. To enable you to quickly evaluate and deploy PGD, we have created this Quick Start so that you can get up and running with a fully configured PGD cluster using the same tools that you will use to deploy to production. 
+PGD is very configurable. To quickly evaluate and deploy PGD, use this quick start. It'll get you up and running with a fully configured PGD cluster using the same tools that you'll use to deploy to production. This quick start includes:
 
-* A short introduction to TPA and how it helps you configure, deploy and manage Postgres Distributed
+* A short introduction to Trusted Postgres Architect (TPA) and how it helps you configure, deploy, and manage Postgres Distributed
 * A guide to selecting Docker or AWS quick starts
   * The Docker quick start 
   * The AWS quick start
+* Connecting applications to your cluster
+* Further explorations with your cluster
+  * Conflicts
+  * Failover
+  * Reconfiguration
 
 
 ## Introducing PGD and TPA
 
-PGD (Postgres Distributed) is a multi-master replicating implementation of Postgres designed for high performance and availability. The installation of PGD is orchestrated by TPA. 
+PGD is a multi-master replicating implementation of Postgres designed for high performance and availability. The installation of PGD is orchestrated by TPA. 
 
-We created TPA (Trusted Postgres Architect) to make the installation and management of various Postgres configurations easily repeatable. TPA orchestrates the creation and deployment of Postgres. 
+We created TPA to make installing and managing various Postgres configurations easily repeatable. TPA orchestrates creating and deploying Postgres. 
 
-In these quick starts, we install TPA first. If you already have TPA installed, you can skip those steps. TPA can be used, and re-used, to deploy various configurations of Postgres clusters.
+In these quick starts, you install TPA first. If you already have TPA installed, you can skip those steps. You can use TPA to deploy various configurations of Postgres clusters.
```

Comment:
I don't think we need to explicitly state that you can use a piece of software for its intended purpose more than once. If you mean something else, please elaborate. If you do need to use the word "reuse,"  it's spelled without a hyphen.

Reply:
That was a requested emphasis, that TPA is not just an installation application for one installation, but can be used for many different installations.

Reply:
I would be more explicit about what you are trying to say in that case. Like a second sentence, something like: You can also use TPA in the future or more than once to ...